Sources/TuistGenerator/Linter/ProjectLinter.swift

```diff
@@ -68,4 +69,28 @@ class ProjectLinter: ProjectLinting {
         }
         return issues
     }
+    
+    private func lintOptions(project: Project) -> [LintingIssue] {
+        var issues: [LintingIssue] = []
+        issues.append(contentsOf: lintNotDuplicatedOptions(project: project))
+        return issues
+    }
+    
+    private func lintNotDuplicatedOptions(project: Project) -> [LintingIssue] {
+        var issues: [LintingIssue] = []
+        let duplicatedOptions = project.options
+            .reduce(into: [ProjectOption: Int]()) { $0[$1] = ($0[$1] ?? 0) + 1 }
```

Comment:
nit:
For the closure, I'd do `$0[$1, defaultValue: 0] += 1` which is IMHO more readable